package.json

```diff
@@ -29,7 +29,10 @@
     "babel-preset-react": "^6.3.13",
     "babel-register": "^6.14.0",
     "css-modules-require-hook": "^4.0.2",
-    "isomorphic-loader": "^1.0.0"
+    "isomorphic-loader": "^1.0.0",
+    "favicons-webpack-plugin": "0.0.7",
+    "sw-precache-webpack-plugin": "^0.5.1",
+    "web-app-manifest-loader": "^0.1.1"
```

Comment:
These ultimately need to go into the archetype, leaving them here for now for ease of development